test/trip_dispatch_test.rb

```diff
@@ -121,5 +127,65 @@ def build_test_dispatcher
         end
       end
     end
+
+    describe "request_trip" do
```

Comment:
You should also include checks to see if the trip end-time is `nil` and that the rating/cost are `nil`.